backend/src/serverless/microservices/nodejs/workerFactory.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-case-declarations */
+import { Edition } from '../../../types/common'
```

Comment:
I moved Edition to @crowd/types library so it should be imported from there AFAIK.